nix/default.nix

```diff
@@ -107,6 +107,7 @@ let
 
     # For cabal-migration
     pkgs.haskellPackages.cabal-plan
+    pkgs.haskellPackages.cabal-fmt
```

Comment:
this doesn't really belong under that comment two lines above, but close enough.

Reply:
Solved it by removning the comment. `cabal-plan` is not just used for migration but for our regular workflows